test/error-placement.js

```diff
@@ -311,6 +311,21 @@ test( "test existing non-error aria-describedby", function( assert ) {
 	strictEqual( "", $("#testForm17text-error").text(), "Error label is empty for valid field" );
 });
 
+test( "test existing non-error aria-describedby with bracket", function( ) {
+	expect( 2 );
+	var form = $( "#testForm18" ),
+		field = $( "#testForm18\\[text\\]");
+
+	field.attr("aria-describedby", "testForm18\\[text\\]-description testForm18\\[text\\]-error");
```

Comment:
I did it to simulate a second validation pass, should have use validator.element() after the form.validate().
ie: Check/Uncheck a checkbox after the initial submit validation have been done will trigger validator.element(), previous error reference could not be found and was added again and again.

The minor fix I provide is to avoid the 'xxx-error' reference to be add multiple time, I actually testing that the reference isn't added again if already there.